src/components/SelfReferee/index.tsx

```diff
@@ -0,0 +1,58 @@
+//nextjs Functional component
+
+import React from "react";
+import Button from "../UI/Button";
+import Content from "../UI/Content";
+import styles from "./SelfReferee.module.css";
+
+const SelfRefree = () => {
+  return (
+    <>
+      <Content title="Self-Refereeing">
+        <div className={styles.selfReferee}>
+          <p>
+            Defifa relies on the integrity of a few transactions made by the
+            game’s participants.
+          </p>
+          <p>
+            Scorecards can be submitted that suggest the correct results of
+            off-chain events.
+          </p>
+          <Button onClick={() => {}} size="big">
+            Submit a scorecard
+          </Button>
+          <br />
+          <br />
+          <p>
+            If you hold an nft, you can send a transaction attesting to a
+            submitted scorecard that conveys correct results of off-chain
+            events.
+          </p>
+          <p className={styles.attestationConfirm}>
+            You’ve attested to scorecard with root <b>0xasdf...1234</b> which
+            matches these results.
```

Comment:
**these results** has an underline, you can wrap it with a span! :)

Reply:
One more thing, all paragraphs in self-referee section should have `font-weight: 500`